scripts/zfs-tests.sh

```diff
@@ -543,8 +549,13 @@ export __ZFS_POOL_EXCLUDE
 export TESTFAIL_CALLBACKS
 export PATH=$STF_PATH
 
-msg "${TEST_RUNNER} ${QUIET} -c ${RUNFILE} -i ${STF_SUITE}"
-${TEST_RUNNER} ${QUIET} -c "${RUNFILE}" -i "${STF_SUITE}"
+TAGSARG=
+if [[ -n $TAGS ]]; then
+	TAGSARG="-T $TAGS"
+fi
```

Comment:
You could simplify this by having `TAGS="functional"` by default.  Then you can unconditionally pass `-T $TAGS` to test-runner to remove the need for `TAGSARG`.